compiler/src/dotty/tools/dotc/tastyreflect/ReflectionInternal.scala

```diff
@@ -28,6 +29,16 @@ class ReflectionCompilerInterface(val rootContext: core.Contexts.Context) extend
   def rootPosition: util.SourcePosition =
     tastyreflect.MacroExpansion.position.getOrElse(SourcePosition(rootContext.source, Spans.NoSpan))
 
+  //
+  // QUOTE UNPICKLING
+  //
+
+  def unpickleExpr(repr: Unpickler.PickledQuote, args: Unpickler.PickledExprArgs): scala.quoted.Expr[_] =
+    new scala.internal.quoted.TastyTreeExpr(PickledQuotes.unpickleExpr(repr, args))
+
+  def unpickleType[T](repr: Unpickler.PickledQuote, args: Unpickler.PickledTypeArgs): scala.quoted.Type[_] =
+    new scala.internal.quoted.TreeType(PickledQuotes.unpickleType(repr, args))
+
```

Comment:
Minor question: what about return `Tree`s instead of `Expr[_]` and `Type[_]`.

Reply:
Because it is specific to Expr and Type. There are no use cases where we need the tree directly.

Reply:
The type parameter `T` is not used.

Reply:
I will remove it